frontend/packages/erd-core/src/features/erd/components/ERDContent/components/TableNode/TableColumnList/TableColumn/TableColumn.tsx

```diff
@@ -12,16 +12,33 @@ type TableColumnProps = {
   handleId: string
   isSource: boolean
   targetCardinality?: CardinalityType | undefined
+  isHovered?: boolean
+  isSelectedTable?: boolean
+  releatedColumns: string[]
 }
 
 export const TableColumn: FC<TableColumnProps> = ({
   column,
   handleId,
   isSource,
   targetCardinality,
+  isHovered,
+  isSelectedTable,
+  releatedColumns,
 }) => {
+  const isRelated = releatedColumns.includes(column.name)
+  const isSelectedOrHovered = (isSelectedTable || isHovered) && isRelated
+  const isHoveredUnselected = isHovered && !isRelated
+
   return (
-    <li key={column.name} className={styles.columnWrapper}>
+    <li
+      key={column.name}
+      className={`
+        ${styles.columnWrapper}
+        ${isSelectedOrHovered ? styles.selectedColumn : ''}
+        ${isHoveredUnselected ? styles.hoveredColumn : ''}
+      `}
```

Comment:
Please use clsx because it contains line breaks.

<img width="1343" alt="Screenshot_2025-05-08_at_15_35_09" src="https://github.com/user-attachments/assets/9603d78b-a99e-41bc-904f-3029a486b405" />



https://github.com/liam-hq/liam/blob/56572e24025c947fe31a1c2ef3f2f43f1015d4b7/frontend/apps/app/components/CommonLayout/GlobalNav/OrganizationItem/OrganizationItem.tsx#L40-L43